cpp/tests/collect_list/collect_list_test.cpp

```diff
@@ -0,0 +1,485 @@
+/*
+ * Copyright (c) 2019-2020, NVIDIA CORPORATION.
```

Comment:
```suggestion
 * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Reply:
Good catch, @sameerz. I'd better check the lot of them.